doc/docker_image.md

```diff
@@ -0,0 +1,168 @@
+# Docker Image for Tapyrus Core
+
+## Quick Start
+
+### Testnet
+
+Refer to [How to start a node on Tapyrus testnet?](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/getting_started.md#how-to-start-a-node-on-tapyrus-testnet)
+
+#### Create `tapyrus.conf`
+
+```bash
+cat << EOS > tapyrus.conf
+networkid=1939510133
+txindex=1
+server=1
+rest=1
+rpcuser=rpcuser
+rpcpassword=rpcpassword
+rpcbind=0.0.0.0
+rpcallowip=127.0.0.1
+addseeder=static-seed.tapyrus.dev.chaintope.com
+EOS
+```
+
+#### Running tapyrus node
+
+```bash
+$ docker run -d --name 'tapyrus_node_testnet' -v $PWD/tapyrus.conf:/etc/tapyrus/tapyrus.conf -e GENESIS_BLOCK_WITH_SIG='01000000000000000000000000000000000000000000000000000000000000000000000044cc181bd0e95c5b999a13d1fc0d193fa8223af97511ad2098217555a841b3518f18ec2536f0bb9d6d4834fcc712e9563840fe9f089db9e8fe890bffb82165849f52ba5e01210366262690cbdf648132ce0c088962c6361112582364ede120f3780ab73438fc4b402b1ed9996920f57a425f6f9797557c0e73d0c9fbafdebcaa796b136e0946ffa98d928f8130b6a572f83da39530b13784eeb7007465b673aa95091619e7ee208501010000000100000000000000000000000000000000000000000000000000000000000000000000000000ffffffff0100f2052a010000002776a92231415132437447336a686f37385372457a4b6533766636647863456b4a74356e7a4188ac00000000' tapyrus/tapyrusd:latest
+
```

Comment:
`-v $PWD/tapyrus.conf:/etc/tapyrus/tapyrus.conf` seems to specify two config files, but entrypoint.sh seems to use only `conf = ${CONF_DIR}/tapyrus.conf`.

Reply:
I don't understand about the two config files. This part specifies only one config file.
`-v $PWD/tapyrus.conf:/etc/tapyrus/tapyrus.conf` tries to mount a conf file on host to container onto the path `/etc/tapyrus/tapyrus.conf`.

Reply:
Oh, I see.